src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -295,6 +295,84 @@ protected Object decode(
                                     value = buf.readByte();
                                     position.set(Position.PREFIX_TEMP + (j + 2 - 45), (value != 0x80) ? value : null);
                                     break;
+                                case 53:
+                                    value = buf.readUnsignedShortLE();
+                                    if (value == 0x7FFF) {
+                                        position.set(Position.KEY_FUEL_LEVEL + "Level", (Boolean) null);
+                                        position.set(Position.KEY_FUEL_LEVEL, (Boolean) null);
+                                    } else if (BitUtil.check(value, 7)) {
+                                        position.set(Position.KEY_FUEL_LEVEL + "Level", BitUtil.to(value, 6));
+                                        position.set(Position.KEY_FUEL_LEVEL, (Boolean) null);
+                                    } else {
+                                        position.set(Position.KEY_FUEL_LEVEL + "Level", (Boolean) null);
+                                        position.set(Position.KEY_FUEL_LEVEL, BitUtil.to(value, 6) / 10);
+                                    }
+                                    break;
+                                case 54:
+                                    position.set(Position.KEY_FUEL_USED, buf.readFloatLE());
+                                    break;
+                                case 55:
+                                    value = buf.readUnsignedShortLE();
+                                    position.set(Position.KEY_RPM, (value != 0xFFFF) ? value : null);
+                                    break;
+                                case 56:
+                                    value = buf.readByte();
+                                    position.set(Position.KEY_COOLANT_TEMP, (value != 0x80) ? value : null);
+                                    break;
+                                case 57:
+                                    position.set(Position.KEY_OBD_ODOMETER, buf.readFloatLE());
+                                    break;
+                                case 58:
+                                case 59:
+                                case 60:
+                                case 61:
+                                case 62:
+                                    value = buf.readUnsignedShortLE();
+                                    position.set(
+                                            Position.KEY_AXLE_WEIGHT + (j + 2 - 58), (value != 65535) ? value : null
+                                    );
```

Comment:
Please keep closing bracket on the previous line... same for other similar places.

Reply:
ok, i'll fix it